src/components/calcite-picker/calcite-picker.tsx

```diff
@@ -38,7 +38,7 @@ export class CalcitePicker {
 
   @Prop({ reflect: true }) editEnabled = false; /* ignored unless mode is configuration */
 
-  @State() selectedValues = new Set();
+  @State() selectedValues = {};
```

Comment:
no more set :(

Would a map work?

Reply:
Can this be typed?

Reply:
> Would a map work?

I think so, will explore it.

> Can this be typed?

It will be a map, with the key will be a string and the value will be a dom object reference. There typing info here you think will add information beyond saying its a map? I don't expect the shape of the dom object to be something we can define.

Reply:
To expedite progress - let's look into using a map on the main PR. The tests should be unaffected. That way we can get the tests merged, which is a blocker for the other PRs right now. Sound good to you guys? @driskull @jcfranco

Reply:
Sounds good.

Reply:
After brief chat with Matt, going to leave as POJO unless we need the ability to track insertion order.